lnclient/ldk/ldk.go

```diff
@@ -485,7 +490,7 @@ func (ls *LDKService) SendPaymentSync(ctx context.Context, invoice string, amoun
 	fee := uint64(0)
 	preimage := ""
 
-	timeout := time.Second * time.Duration(timeoutSeconds)
+	timeout := time.Second * time.Duration(*timeoutSeconds)
```

Comment:
this will crash

Reply:
Why? I tested by replacing 50 above with 1 and it worked for me on LDK (paid refund@lnurl.mutinynet.com)